src/App.js

```diff
@@ -51,8 +51,7 @@ class AppFrame extends Component {
 
   onSignIn(status, data) {
     this.setState({loggedIn: status, userData: data});
-    data.basic.username == "adminacc" ? this.setState({admin: true}) : this.setState({admin: false});
-      
+    if (data.basic.username == "adminacc") {this.setState({admin: true})};
```

Comment:
We should change this to accept an admin status as a part of the user data object. This should be a boolean value as opposed to a hard coded string.

Reply:
done